internal/manifests/queryfrontend/query_frontend.go

```diff
@@ -366,7 +366,6 @@ func enableMonitoringTab(tempo v1alpha1.TempoStack, jaegerQueryContainer corev1.
 			},
 		},
 		Args: []string{
-			"--prometheus.query.support-spanmetrics-connector",
```

Comment:
Should the env var be set somewhere instead of the cli arg?

Reply:
It should be no longer needed. Previously it was already enabled by default.